torch/onnx/utils.py

```diff
@@ -445,6 +447,24 @@ def _get_named_param_dict(graph, params):
     _params_dict = dict(zip(param_names, params))
     return _params_dict
 
+def _get_example_outputs(model, args):
+    input_args = copy.deepcopy(args)
+    input_kwargs = {}
+    if input_args and isinstance(input_args[-1], dict):
+        input_kwargs = input_args[-1]
+        input_args = input_args[:-1]
+    try:
+        model_copy = copy.deepcopy(model)
```

Comment:
Can we do deepcopy only if `training is True`?

Reply:
Make sense. I removed deepcopy here. Thanks!